nixos/modules/services/networking/nebula.nix

```diff
@@ -84,6 +84,24 @@ in
                 description = "Whether this node is a relay.";
               };
 
+              lighthouse.serveDns = lib.mkOption {
+                type = lib.types.bool;
+                default = false;
+                description = "Whether this lighthouse node should serve DNS.";
+              };
+
+              lighthouse.dns.host = lib.mkOption {
+                type = lib.types.str;
+                default = "0.0.0.0";
```

Comment:
Better default is possibly localhost.

Reply:
I agree `localhost` is better as a "secure defaults" choice, so the default doesn't listen on public interfaces. The description may want a callout, though, or have an example value of some nebula address like "10.0.0.5".